pkg/controller/backup/backup_controller.go

```diff
@@ -201,6 +284,64 @@ func (r *ReconcileBackup) Reconcile(request reconcile.Request) (reconcile.Result
 	return reconcile.Result{}, nil
 }
 
+// DeleteResticSnapshot creates the job to forget a restic snapshot.
+func (r *ReconcileBackup) DeleteResticSnapshot(backup *extensionv1.Backup) error {
+	var params = resticutils.PodSpecParams{
+		CPU:         "100m",
+		Memory:      "512Mi",
+		ResticImage: "docker.io/restic/restic:0.9.5",
+		MySQLImage:  "skpr/mtk-mysql",
+		WorkingDir:  "/home/shepherd",
+		Tags:        []string{},
+	}
+
+	spec, err := resticutils.PodSpecDelete(
+		backup.Status.ResticID,
+		backup.ObjectMeta.Namespace,
+		backup.ObjectMeta.GetLabels()["site"],
+		params,
+	)
+	if err != nil {
+		return err
+	}
+
+	var (
+		parallelism    int32 = 1
+		completions    int32 = 1
+		activeDeadline int64 = 3600
+		backOffLimit   int32 = 2
+		//ttl            int32 = 3600
+	)
+
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-delete-%s", resticutils.Prefix, backup.Name),
+			Namespace: backup.ObjectMeta.Namespace,
+			Labels: map[string]string{
+				"app":          "restic",
+				"resticAction": "delete",
+			},
+		},
+		Spec: batchv1.JobSpec{
+			// @todo uncomment this when the feature becomes available (requires kube v1.12+).
+			// ttlSecondsAfterFinished: &ttl,
```

Comment:
When UoA's api version is updated to 1.12+ then this should be uncommented. It will automatically delete jobs after 1 hour.

Currently these delete jobs hang around indefinitely but could be deleted by using label selectors.

```
 oc delete job --selector=resticAction=delete
```

Reply:
Doh. That won't happen till we move to OpenShift 4.x.

We'll just periodically run it manually :+1: